packages/aws-cdk/lib/cli/cli.ts

```diff
@@ -107,7 +108,7 @@ export async function exec(args: string[], synthesizer?: Synthesizer): Promise<n
       proxyAddress: argv.proxy,
       caBundlePath: argv['ca-bundle-path'],
     },
-    logger: new SdkToCliLogger(ioHost),
+    logger: new SdkToCliLogger(asIoHelper(ioHost, ioHost.currentAction as any)),
```

Comment:
`ioHost.currentAction` is extending `ToolkitAction` with additional values. This is fine.
Same again in L189